src/hyperfine/timer/unix_timer.rs

```diff
@@ -63,6 +64,7 @@ fn cpu_time_interval(start: &CPUTimes, end: &CPUTimes) -> CPUInterval {
     CPUInterval {
         user: ((end.user_usec - start.user_usec) as f64) * 1e-6,
         system: ((end.system_usec - start.system_usec) as f64) * 1e-6,
+        max_rss: ((end.max_rss_byte - start.max_rss_byte) as f64) * 1e-6,
```

Comment:
I would prefer if we used kibibytes and mebibytes here, i.e. divide by 1024 or 1024². All references of `MegaBytes` should then be changed to `KibiBytes` or `MebiBytes`. In the output we would ideally use "KiB" or "MiB".

This is to avoid the confusion with "KB" and "MB" where it's unclear if they refer to the SI standard (as they should, in my opinion), or if it's an old Unix program which uses "KB" and "MB" when they actually refer to "1024 byte" or "1024² byte".

Reply:
Sounds correct.